codexctl/device.py

```
@@ -161,13 +160,13 @@ def connect_to_device(
             remote_address = self.get_remarkable_address()
             self.address = remote_address # For future reference
         else:
-            if self.check_is_address_reachable(remote_address) is False:
+            if self.check_if_address_reachable(remote_address, port) is False:
                 raise SystemError(f"Error: Device {remote_address} is not reachable!")
 
         client = paramiko.client.SSHClient()
         client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
 
-        if authentication:
+        if authentication != None:
```

Ruff (0.8.0)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)